### DISCUSSION TITLE
1 şubatta herkes ibaxtan yararlanacak mı ?

### DISCUSSION THREAD

**Discussion by Deadeyetr46** [IBAX-io/go-ibax#707] [Soru Cevap - Türkçe]
Soru açık aklıma takıldı bu

**Comment by Yusufknk** [comment]
Evet, herkes topladığı puanlara göre coin alacaklar.

**Reply by Azizatli21** [reply]
Nasıl satacaz

**Reply by Yusufknk** [reply]
Henüz airdrop aşamasındayız, ardından ön satış ve borsa listelenme işlemleri olacak. Gelişmeleri sosyal medya kanallarından takip edebilirsiniz.

**Comment by canrasim** [comment]
bence herkez yararlanacak ama biraz daha eleyecek durumlar olsa
